config/initializers/load_project_configs.rb

```diff
@@ -1,3 +1,4 @@
-FEDORA_CONFIG_GLOBAL = YAML.load_file(Rails.root.join('config', 'fedora.yml'))[Rails.env]
-DERIVATIVE_CONFIG_GLOBAL = YAML.load_file(Rails.root.join('config', 'derivatives.yml'))[Rails.env]
-IIIF_SERVER = YAML.load_file(Rails.root.join('config', 'iiif_server.yml'))[Rails.env]
\ No newline at end of file
+# frozen_string_literal: true
```

Comment:
Let's delete this file; we'll set the values for these using `.env`, since that's more in line with how we're handling this for other projects.